### ISSUE TITLE
Not able to detect mock location

### ISSUE THREAD

**Issue by anantraj2808** [zeexan-dev/detect_fake_location#2]
This package is not able to detect mock location in case of "Single Spot Movement"/"Walkthrough" mode. Kindly let me know how to counter that.

**Comment by sadaqatdev** [comment]
same here

**Comment by zeexan-dev** [comment]
> This package is not able to detect mock location in case of "Single Spot Movement"/"Walkthrough" mode. Kindly let me know how to counter that.

@anantraj2808 @sadaqatdev . Thanks for the feedback. Can you please explain more that from which software you are trying to mock the location

**Comment by waihanko** [comment]
`https://play.google.com/store/apps/details?id=com.lexa.fakegps&hl=en`

Not working for this mock app

**Comment by zeexan-dev** [comment]
True. Currently, detecting mock locations is not possible on Android 10+. While this is a known limitation, I will continue to monitor for any future changes or updates that may allow this functionality to be restored.

**Comment by zeexan-dev** [comment]
> `https://play.google.com/store/apps/details?id=com.lexa.fakegps&hl=en`
> 
> Not working for this mock app

True. Currently, the package is not detecting the mock location. I'm trying to find the solution and will update it asap.

**Comment by zeexan-dev** [comment]
@waihanko the issue is resolved. Thanks.

**Comment by zeexan-dev** [comment]
2ea3e3e8ee8cf108024184e3f74156effdc31b66
